1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -24,7 +24,11 @@ let shapes = {
 let shapesWithMoreThanFiveSides = [];
 
 // WRITE YOUR CODE BELOW THIS LINE
-
+for (let sides in shapes) {
```

Comment:
usually the variable names in the loop is singular to show iteration over each item in the list